src/evict/evict_page.c

```diff
@@ -125,6 +125,7 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
      */
     time_start = 0;
     if (LF_ISSET(WT_EVICT_CALL_URGENT)) {
+        ++ref->page->page_evict_count;
```

Comment:
Why not use `cache_eviction_force`?

Reply:
I got this from your prototype change. I'll change to use cache_eviction_force.